packages/services/src/nbconvert/index.ts

```diff
@@ -28,9 +30,11 @@ export class NbConvertManager {
   readonly serverSettings: ServerConnection.ISettings;
 
   /**
-   * Get whether the application should be built.
+   * Fetch and cache the export formats from the expensive nbconvert handler.
    */
-  async getExportFormats(): Promise<NbConvertManager.IExportFormats> {
+  async fetchExportFormats(): Promise<NbConvertManager.IExportFormats> {
```

Comment:
Do we need this to be public?

Reply:
Not strictly.

Reply:
Made `protected`, so that it can still be overridden without hacks.